app/main.py

```diff
@@ -254,14 +247,78 @@ def change_zoom_center(n_clicks=None):
     return define_map_zoom_center(n_clicks, alert_metadata)
 
 
+@app.callback(
+    [Output('img_url', 'children'), Output('live_alert_header_btn', 'children'),
+     Output('live_alerts_marker', 'children')],
+    Input('interval-component', 'n_intervals'))
+def fetch_alert_status_metadata(n_intervals):
+    '''
+    This callback takes as input the interval-component which acts as a timer,
+    scheduling API metadata fetches and defining alert status
+    '''
+    # Fetching live alerts where is_acknowledged is False
+    response = api_client.get_ongoing_alerts().json()
+    all_alerts = pd.DataFrame(response)
+    live_alerts = all_alerts.loc[all_alerts['is_acknowledged'] == False]
```

Comment:
```suggestion
    live_alerts = all_alerts.loc[~all_alerts['is_acknowledged']]
```

Flake8 doesn't like this `== False` apparently, I just proposed another version that should work the same and satisfy styling requirements.

Reply:
Thanks, done